src/connector/src/datagen/source/field_generator/mod.rs

```diff
@@ -12,72 +12,93 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+mod timestamp;
+mod varchar;
+
 use anyhow::Result;
 use rand::{thread_rng, Rng};
 use risingwave_common::types::DataType;
 use serde_json::{json, Value};
+pub use timestamp::*;
+pub use varchar::*;
+
+pub trait FieldGenerator {
```

Comment:
How about `NumericFieldGenerator`? I guess it cannot be applied to Timestamp/VARCHAR. And better to add some comments for pub trait.